source/index.html.md

```diff
@@ -219,6 +219,8 @@ error categories, the last two digits define specific errors.
 * 0708: Max attempts reached to perform operation
 * 0709: Action not permitted for missing KYC data
 * 0710: Password must be updated
+* 0711: There was an error trying to save the users business persons
```

Comment:
Can't be used.. is already in use by this PR - https://github.com/bitsoex/slate/pull/77